external/concealer/package.xml

```diff
@@ -13,6 +13,7 @@
   <!-- Autoware.Universe -->
   <depend>autoware_adapi_v1_msgs</depend>
   <depend>autoware_control_msgs</depend>
+  <depend>autoware_internal_msgs</depend>
```

Comment:
Please specify autoware_internal_planning_msgs.
Otherwise, it fails to find the message.

Reply:
```suggestion
  <depend>autoware_internal_planning_msgs</depend>
```

Reply:
Done.